### DISCUSSION TITLE
You can add a code and preview switching that can be added to the control deletion function.

### DISCUSSION THREAD

**Discussion by muzihuaner** [tyron12233/CodeAssist#297] [Ideas]
You can add a code and preview switching that can be added to the control deletion function.

这里能不能添加一个代码和预览切换的功能还有控件删除功能
![Screenshot_2022-03-02-09-03-59-43_2870cb4d7e72f8050b62d08052357b72](https://user-images.githubusercontent.com/20607912/156275490-91210ff2-75e0-4069-ad63-9f6e2636be4d.jpg)

**Comment by yvvt0379** [comment]
Feature request:
1. Switching between code editing and layout previewing.
2. Deleting widgets.

**Comment by tyron12233** [comment]
1. You can switch back to the code editor from layout preview by pressing the back button.
2. You can delete widgets by dragging it outside the editor.

**Comment by tyron12233** [comment]
On future requests, use the [discussion tab](https://github.com/tyron12233/CodeAssist/discussions) to prioritize issues.
